src/shapes/object.class.js

```diff
@@ -645,6 +645,11 @@
      */
     absolutePositioned: false,
 
+    /**
+     * Indicates whether this object can be erased by @class fabric.EraserBrush
+     */
+    erasable: true,
```

Comment:
Let's add this in the mixin file, where getEraser is, and let's add proper JSDOC ( with @type )

Reply:
DONE